src/components/FiatValue.vue

```diff
@@ -34,7 +33,10 @@ export default {
     },
     fiatValue() {
       if (!this.rate) return null;
-      if (this.token && !this.tokenData) return null;
+      if (this.token && !this.tokenData) {
+        return new BigNumber(0).toNumber()
+          .toLocaleString('en-US', { style: 'currency', currency: this.showCurrency });
```

Comment:
```suggestion
        return (0).toLocaleString('en-US', { style: 'currency', currency: this.showCurrency });
```
BigNumber is not needed to do this 🤔